exovetter/vetters.py

```diff
@@ -3,6 +3,11 @@
 from abc import ABC, abstractmethod
 
 from exovetter import lpp
+from exovetter import odd_even
+from exovetter import transit_coverage
+
+import astropy.units as u
+import exovetter.const as const
```

Comment:
Astropy likes to use `const` too, so if we use `exo_const`, it would be less confusing if in the future we also use `astropy.constants`.
```suggestion
from exovetter import const as exo_const
```